examples/stress_tests/many_buttons.rs

```diff
@@ -64,50 +89,119 @@ fn button_system(
+
+    let buttons_f = args.buttons as f32;
+    let border = if args.no_borders {
+        UiRect::ZERO
+    } else {
+        UiRect::all(Val::VMin(0.05 * 90. / buttons_f))
+    };
+
+    let as_rainbow = |i: usize| Color::hsl((i as f32 / buttons_f) * 360.0, 0.9, 0.8);
+    commands.spawn(Camera2dBundle::default());
+    commands
+        .spawn(NodeBundle {
+            style: Style {
+                flex_direction: FlexDirection::Column,
+                justify_content: JustifyContent::Center,
+                align_items: AlignItems::Center,
+                width: Val::Percent(100.),
+                ..default()
+            },
+            ..default()
+        })
+        .with_children(|commands| {
+            for column in 0..args.buttons {
+                commands
+                    .spawn(NodeBundle::default())
+                    .with_children(|commands| {
+                        for row in 0..args.buttons {
+                            let color = as_rainbow(row % column.max(1)).into();
+                            let border_color = Color::WHITE.with_a(0.5).into();
+                            spawn_button(
+                                commands,
+                                color,
+                                buttons_f,
+                                column,
+                                row,
+                                !args.no_text,
+                                border,
+                                border_color,
+                                image
+                                    .as_ref()
+                                    .filter(|_| (column + row) % args.image_freq == 0)
+                                    .cloned(),
+                            );
+                        }
+                    });
+            }
+        });
+}
+
+fn setup_grid(mut commands: Commands, asset_server: Res<AssetServer>, args: Res<Args>) {
+    println!("grid setup");
     warn!(include_str!("warning_string.txt"));
+    let image = if 0 < args.image_freq {
+        Some(asset_server.load("branding/icon.png"))
+    } else {
+        None
+    };
+
+    let buttons_f = args.buttons as f32;
+    let border = if args.no_borders {
+        UiRect::ZERO
+    } else {
+        UiRect::all(Val::VMin(0.05 * 90. / buttons_f))
+    };
 
-    let count = ROW_COLUMN_COUNT;
-    let count_f = count as f32;
-    let as_rainbow = |i: usize| Color::hsl((i as f32 / count_f) * 360.0, 0.9, 0.8);
+    let as_rainbow = |i: usize| Color::hsl((i as f32 / buttons_f) * 360.0, 0.9, 0.8);
     commands.spawn(Camera2dBundle::default());
     commands
         .spawn(NodeBundle {
             style: Style {
+                display: Display::Grid,
                 width: Val::Percent(100.),
+                height: Val::Percent(100.0),
+                grid_template_columns: vec![RepeatedGridTrack::flex(args.buttons as u16, 1.0)],
+                grid_template_rows: vec![RepeatedGridTrack::flex(args.buttons as u16, 1.0)],
```

Comment:
```suggestion
                grid_template_columns: RepeatedGridTrack::flex(args.buttons as u16, 1.0),
                grid_template_rows: RepeatedGridTrack::flex(args.buttons as u16, 1.0),
```

`Vec<RepeatedGridTrack>` has `From<RepeatedGridTrack>`